snippets/distance.md

```diff
@@ -0,0 +1,36 @@
+### distance
+
+Returns distance between two lat/long points in different units.
+
+Use a trigonometry formulas to count distance between two geo locations and converts into standard units.
+```
+"M" :   Miles
+"K" :   KiloMeter
+"N" :   Nautical Miles
+```
+
+
+```php
+function distance($lat1, $lon1, $lat2, $lon2, $unit = 'M')
+{
+    $miles = rad2deg(acos(sin(deg2rad($lat1)) * sin(deg2rad($lat2)) + cos(deg2rad($lat1)) * cos(deg2rad($lat2)) * cos(deg2rad($lon1 - $lon2)))) * 60 * 1.1515;
```

Comment:
What distance approach do you use? Haversine or Great-circle formula

Can you explain about that?